City-of-Helsinki/drupal-helfi-platform-config
# UHF-3169: Changed contact persons title to job title

# [UHF-3169](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3169)

Fixed issue where contact person job title translation would go to wrong places and the job title itself got the wrong Finnish translation

## What was done
* Changed field_contact_title label "Title" -> "Job title"

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `make shell` in project root
    * `composer require drupal/helfi_platform_config:dev-UHF-3169-field-label-change`
    * `drush cr; drush updb`

## How to test
* Add the contact listing paragraph to a standard page, when the admin panel is in English the contact persons field_contact_title should now have the label "Job title" and if the admin panel is in Finnish it should have "Titteli"
* Also check that when adding other paragraphs that have a title field, the translation in Finnish is still Otsikko and not Titteli
